contracts/sources/obligation.move

```diff
@@ -379,6 +481,12 @@ module suilend::obligation {
         clock: &Clock,
         max_repay_amount: Decimal,
     ): Decimal {
+        let borrow_weight = if (is_emode(obligation)) {
```

Comment:
nit: move this line closer to where the value is used, ie right before 

` if (le(interest_diff, repay_amount)) {`